build/FrameworkPackage/MakeFrameworkPackage.ps1

```diff
@@ -74,40 +65,35 @@ ForEach ($input in ($inputs -split ";"))
     Write-Verbose "Copying $inputBasePath\Themes"
     Copy-IntoNewDirectory -IfExists $inputBasePath\Themes $fullOutputPath\PackageContents\Microsoft.UI.Xaml
 
+    $sdkReferencesPath=$kitsRoot10+"\References\10.0.18323.0";
+    $wimdReferences = $sdkReferencesPath + "\Windows.Foundation.FoundationContract\3.0.0.0\Windows.Foundation.FoundationContract.winmd;" + $sdkReferencesPath + "\Windows.Foundation.UniversalApiContract\8.0.0.0\Windows.Foundation.UniversalApiContract.winmd"
+    $classes = Get-WinmdTypes $inputBasePath\$inputBaseFileName.winmd  $wimdReferences
+
 @"
 "$inputBaseFileName.dll" "$inputBaseFileName.dll"
-"$inputBaseFileName.winmd" "$inputBaseFileName.winmd"
+"$inputBaseFileName.winmd" "$inputBaseFileName.winmd" 
 "@ | Out-File -Append -Encoding "UTF8" $fullOutputPath\PackageContents\FrameworkPackageFiles.txt
 
-    $ActivatableTypes += $activatableTypeEntries["$($inputBaseFileName.ToLower()).dll"]
-
-    # NOTE: Build machines don't have ILDAsm or .NET SDK so we have to do this differently.
-#    # Get the activatable types out of each WinMD
-#    $ildasmCommand = "`"ildasm.exe`" /classlist `"$inputBasePath\$inputBaseFileName.winmd`" /text"
-#    Write-Host "cmd /c $ildasmCommand"
-#    $classes = cmd /c $ildasmCommand | where {$_.Contains(".class public")}
-#    $classes = $classes -replace '.* ([\w\.]+)$',"`$1"
+    $ActivatableTypes += @"
+    <Extension Category="windows.activatableClass.inProcessServer">
+      <InProcessServer>
+        <Path>$inputBaseFileName.dll</Path>
 
-#    $ActivatableTypes += @"
-#    <Extension Category="windows.activatableClass.inProcessServer">
-#      <InProcessServer>
-#        <Path>$inputBaseFileName.dll</Path>
-
-#"@
-#    ForEach ($class in $classes)
-#    {
-#        Write-Host "Activatable type : $class"
-#        $ActivatableTypes += "        <ActivatableClass ActivatableClassId=`"$class`" ThreadingModel=`"both`" />`r`n"
-#    }
+"@
+    ForEach ($class in $classes)
+    {
```

Comment:
It's probably fine (in that it isn't technically wrong) to just grab all classes but I suspect this list is much longer than the old one -- I think these classes include interfaces, enums and non-activatable types. Can you try to get the list to match the old one?

Reply:
i filtered it some but i'm not sure how to get the exact same list. It looks like it is filtering out attributes, delegates etc.

Reply:
You can probably find the code for the task in GitHub or maybe internal code search? But I'd just do some guessing. Delegates aren't activatable, so they don't need to be in there. I would guess that you could key off the presence of the Threading or MarshalingBehavior attribute?

Reply:
I'm using the Activatable, Composable and Statics attributes to filter the list which matches exactly what we have today. Except of course the private apis.